api/src/main/java/ca/bc/gov/educ/api/graduation/config/RequestInterceptor.java

```diff
@@ -68,6 +81,8 @@ public void afterCompletion(@NonNull final HttpServletRequest request, final Htt
 		val correlationID = request.getHeader(EducGraduationApiConstants.CORRELATION_ID);
 		if (correlationID != null) {
```

Comment:
Here the username and request source will only be added as headers if correlationid is not null. This makes no sense. Again, remove the if statement and use ternary statements to set the headers.